packages/core/android/src/test/kotlin/com/datadog/tools/unit/GenericAssert.kt

```diff
@@ -22,6 +22,17 @@ class GenericAssert(actual: Any) :
         return this
     }
 
+    fun <T> getField(name: String, nestedAssert: (T) -> Unit = {}): GenericAssert {
```

Comment:
I think we don't need lambda as argument. We can just return `field` from this method instead of `GenericAssert` and use that value for assertions. And this method can be called `getActualValue` (currently the return type doesn't really match the name of the method).